longestString/longestString.js

```diff
@@ -1,4 +1,12 @@
 /*
  * Write a function that accepts an array of strings.
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+
+  const longestString = (array) => {
+    let longest = array[0].length.
```

Comment:
I love how you started on the second index, and came into the the function with having discovered the first index off the bat! Great job!